recipes/isl/all/conanfile.py

```diff
@@ -39,54 +39,61 @@ def configure(self):
         del self.settings.compiler.libcxx
         del self.settings.compiler.cppstd
 
+        if self.settings.os == "Windows" and self.options.shared:
```

Comment:
it can be promoted to validate() method:

https://blog.conan.io/2020/12/14/New-conan-release-1-32.html

Reply:
This is rarely being done and I needs a hook... is there a related issue? I don't see it https://github.com/conan-io/hooks/issues

Reply:
It is too new and there might be valid cases where you want to throw earlier, not sure if a hook is the answer

Reply:
Unless a check requires to access the final version/options values of upstreams, I don't see the advantage to check in `validate()`

Reply:
It's starting to get messy when/why to use this new feature... we need to be more vigilant when reviewing